client/Windows/wf_graphics.c

```diff
@@ -130,6 +130,12 @@ static void wf_Bitmap_Free(rdpContext* context, rdpBitmap* bitmap)
 		SelectObject(wf_bitmap->hdc, wf_bitmap->org_bitmap);
 		DeleteObject(wf_bitmap->bitmap);
 		DeleteDC(wf_bitmap->hdc);
+
+		if (wf_bitmap->_bitmap.data)
+		{
```

Comment:
good catch

Reply:
heh - not exactly a catch. my client plays back video and each successive playback was 25% or more slower than the last. only then did I see the memory growing